recipes/pvxslibs/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pvxslibs" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 46b0c31aa8e47813dc409d58067acf637650cdcf2ed415cc2c3a8cecf8231e4d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Please read the entire build log. It looks like your compilation is also failing due to missing header files.

Reply:
Unless I'm missing the message you're referring to, the indicated missing header files are not actually in error here. This, for example, is checking the build host for what exists on the system and what can/cannot compile:

```
2023-10-03T23:03:05.4616240Z   /tmp/tmpaqzexwgc/try_compile.c:28:10: fatal error: 'sys/sendfile.h' file not found
2023-10-03T23:03:05.4617840Z   #include <sys/sendfile.h>
2023-10-03T23:03:05.4620970Z            ^~~~~~~~~~~~~~~~
2023-10-03T23:03:05.4623030Z   1 error generated.
2023-10-03T23:03:05.4656180Z   Probe includes ['sys/sendfile.h'] -> Absent
```
It's effectively a `./configure` step.

Reply:
Got it.